changelog/fix_fix_a_false_positive_for.md

```diff
@@ -0,0 +1 @@
+* [#11151](https://github.com/rubocop/rubocop/pull/11151): Fix a false positive for `Lint/SuppressedException`. ([@akihikodaki][https://github.com/akihikodaki])
```

Comment:
Can you update it?

```suggestion
* [#11151](https://github.com/rubocop/rubocop/pull/11151): Fix a false positive for `Lint/SuppressedException` when using numbered parameter. ([@akihikodaki][])
```

Reply:
Done with commit b9c2d5e3268695faeb3cb26114fe18e17823aa26.